src/main/java/http_server/ClientSocket.java

```diff
@@ -4,7 +4,11 @@
 import java.io.OutputStream;
 
 public interface ClientSocket {
+
     InputStream getInputStream();
+
     OutputStream getOutputStream();
+
     void close();
-}
+
+}
```

Comment:
I like that you've made the whitespace consistent 👍. I see that there's a separate commit with (hopefully) all of the newline / whitespace cleanup. I'd suggest keeping those kinds of commits out of future PRs as it makes it more difficult to see the actual changes. I'd personally be fine with a commit directly to master with formatting changes, or a separate PR if you really want to still go that route.